packages/panzer/mini-em/example/BlockPrec/CMakeLists.txt

```diff
@@ -63,23 +63,23 @@ TRIBITS_ADD_TEST(
    NUM_MPI_PROCS 4
    )
 
-IF(NOT Kokkos_ENABLE_Cuda AND NOT Kokkos_ENABLE_OpenMP AND MueLu_ENABLE_Epetra)
-
-#  TRIBITS_ADD_TEST(
-#    BlockPrec
-#    NAME "MiniEM-BlockPrec_Augmentation_Epetra"
-#    ARGS "--solver=Augmentation --numTimeSteps=1 --linAlgebra=Epetra"
-#    COMM serial mpi
-#    NUM_MPI_PROCS 1
-#    )
-
-#  TRIBITS_ADD_TEST(
-#    BlockPrec
-#    NAME "MiniEM-BlockPrec_Augmentation_Epetra"
-#    ARGS "--solver=Augmentation --numTimeSteps=1 --linAlgebra=Epetra"
-#    COMM mpi
-#    NUM_MPI_PROCS 4
-#    )
+IF(NOT Kokkos_ENABLE_Cuda AND NOT Kokkos_ENABLE_OpenMP AND MueLu_ENABLE_Epetra AND NOT HAVE_TPETRA_INST_INT_LONG_LONG)
```

Comment:
Wouldn't you need to say `AND HAVE_TPETRA_INST_INT_INT` instead?  Tpetra has other options for GlobalOrdinal, like `long`.

Reply:
True. Is there a way to check from cmake what Tpetra's default GlobalOrdinal will be?

Reply:
> Is there a way to check from cmake what Tpetra's default GlobalOrdinal will be?

Alas no, not yet, but that will change :-) In this case, since you're running Epetra through MueLu (presumably), you really do need `GlobalOrdinal=int`.